LibreNMS/OS/ArubaInstant.php

```diff
@@ -111,7 +111,7 @@ public function discoverWirelessClients()
             // Clients Per SSID
             foreach ($ssid_data as $index => $entry) {
                 $combined_oid = sprintf('%s::%s.%s', $ai_mib, 'aiSSIDClientNum', $index);
-                $oid = snmp_translate($combined_oid, 'ALL', 'arubaos', '-On', null);
+                $oid = snmp_translate($combined_oid, 'ALL', 'arubaos', '-On', $this->getDeviceArray());
```

Comment:
device is not required for snmp_translate().

Should either omit the 'arubaos' mib directory or the device argument.

Reply:
```php
>>> snmp_translate('foobar'', 'ALL', 'arubaos', '-On', null);
<warning>PHP Notice:  Trying to access array offset on value of type null in librenms/includes/snmp.inc.php on line 50</warning>
<warning>PHP Notice:  Trying to access array offset on value of type null in librenms/includes/snmp.inc.php on line 51</warning>
<warning>PHP Notice:  Trying to access array offset on value of type null in librenms/includes/snmp.inc.php on line 68</warning>
```

https://github.com/librenms/librenms/blob/40517f695a9aa8121df9eea3f80f022650644d67/includes/snmp.inc.php#L46-L73

Reply:
The default for the device argument is [], just delete the stupid null.

Reply:
I fixed it better instead